src/Files.App/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -1,210 +1,131 @@
-									<MenuFlyoutItem.Icon>
-										<FontIcon Glyph="&#xE737;" />
-									</MenuFlyoutItem.Icon>
-								</MenuFlyoutItem>
-								<MenuFlyoutItem
-									x:Name="PinToFavorites"
-									Click="PinToFavorites_Click"
-									DataContext="{x:Bind Item}"
-									Text="{helpers:ResourceString Name=BaseLayoutItemContextFlyoutPinToFavorites/Text}"
-									Visibility="Collapsed">
-									<MenuFlyoutItem.Icon>
-										<FontIcon Glyph="&#xE840;" />
-									</MenuFlyoutItem.Icon>
-								</MenuFlyoutItem>
-								<MenuFlyoutItem
-									x:Name="UnpinFromFavorites"
-									Click="UnpinFromFavorites_Click"
-									DataContext="{x:Bind Item}"
-									Text="{helpers:ResourceString Name=BaseLayoutContextFlyoutUnpinFromFavorites/Text}"
-									Visibility="Collapsed">
-									<MenuFlyoutItem.Icon>
-										<FontIcon Glyph="&#xE77A;" />
-									</MenuFlyoutItem.Icon>
-								</MenuFlyoutItem>
-								<MenuFlyoutItem
-									x:Name="PropertiesItem"
-									Click="OpenDriveProperties_Click"
-									DataContext="{x:Bind Item}"
-									Text="{helpers:ResourceString Name=DrivesWidgetContextFlyoutPropertiesMenuItem/Text}">
-									<MenuFlyoutItem.Icon>
-										<FontIcon Glyph="&#xE946;" />
-									</MenuFlyoutItem.Icon>
-								</MenuFlyoutItem>
-								<MenuFlyoutItem
-									x:Name="EjectDevice"
-									x:Load="{x:Bind Item.IsRemovable, Mode=OneWay}"
-									Click="EjectDevice_Click"
-									DataContext="{x:Bind Item}"
-									Text="{helpers:ResourceString Name=SideBarEjectDevice/Text}">
-									<MenuFlyoutItem.Icon>
-										<FontIcon Glyph="&#xF847;" />
-									</MenuFlyoutItem.Icon>
-								</MenuFlyoutItem>
-								<MenuFlyoutItem
-									x:Name="DisconnectNetworkDrive"
-									x:Load="{x:Bind Item.IsNetwork, Mode=OneWay}"
-									Click="DisconnectNetworkDrive_Click"
-									DataContext="{x:Bind Item}"
-									Text="{helpers:ResourceString Name=SideBarDisconnectNetworkDrive/Text}">
-									<MenuFlyoutItem.Icon>
-										<FontIcon Glyph="&#xE8CD;" />
-									</MenuFlyoutItem.Icon>
-								</MenuFlyoutItem>
-							</MenuFlyout.Items>
-						</MenuFlyout>
-					</Button.ContextFlyout>
-					<Grid
-						Margin="8"
-						HorizontalAlignment="Stretch"
-						VerticalAlignment="Stretch"
-						ColumnSpacing="8">
-						<Grid.ColumnDefinitions>
-							<ColumnDefinition Width="32" />
-							<ColumnDefinition Width="*" />
-							<ColumnDefinition Width="Auto" />
-						</Grid.ColumnDefinitions>
-						<Grid.RowDefinitions>
-							<RowDefinition Height="*" />
-							<RowDefinition Height="*" />
-							<RowDefinition Height="*" />
-						</Grid.RowDefinitions>
-						<Image
-							Grid.RowSpan="3"
-							Grid.Column="0"
-							HorizontalAlignment="Center"
-							VerticalAlignment="Center"
-							x:Phase="1"
-							Source="{x:Bind Thumbnail, Mode=OneWay}"
-							Stretch="Uniform" />
+    x:Class="Files.App.UserControls.Widgets.DrivesWidget"
```

Comment:
This is wrong, spacing should be set to tabs.

Reply:
I used XAML styler, how should I configure it?